src/Petabridge.Templates.csproj

```diff
@@ -2,21 +2,18 @@
   <PropertyGroup>
     <PackageType>Template</PackageType>
     <PackageReadmeFile>README.md</PackageReadmeFile>
-    <PackageVersion>2.0.0</PackageVersion>
+    <PackageVersion>2.0.1</PackageVersion>
     <PackageId>Petabridge.Templates</PackageId>
     <Title>Petabridge.Templates</Title>
     <Authors>Petabridge</Authors>
     <Description>Professional .NET Core templates complete with CI, Docs, and more. Supports library, Akka.NET, and ASP.NET Core application types.</Description>
     <PackageTags>dotnet-new;templates;petabridge;akka;</PackageTags>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageReleaseNotes>- Switched build automation system from FAKE to [NUKE](https://nuke.build/)
-- Upgraded [pb-lib](https://github.com/petabridge/Petabridge.Library/), [`pb-akka-cluster`](https://github.com/petabridge/Petabridge.App) and [`pb-akka-web`](https://github.com/petabridge/Petabridge.App.Web) to all use .NET 6
-- Replaced `common.props` with `Directory.Build.props`
-- Git versioning with [GitVersion](https://gitversion.net/)
-- Added README to NuGet Package</PackageReleaseNotes>
+    <PackageReleaseNotes>- Fix missing `.gitignore` file - set `NoDefaultExcludes` to `true`.</PackageReleaseNotes>
     <IncludeContentInPack>true</IncludeContentInPack>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <ContentTargetFolders>content</ContentTargetFolders>
+    <NoDefaultExcludes>true</NoDefaultExcludes>
```

Comment:
Can we make sure that the `.git` folder in each of these template submodules is ignored? that would still be useful.

Reply:
Latest commit should resolve this.